Library/Formula/cctools.rb

```diff
@@ -11,6 +11,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation
     cellar :any
```

Comment:
Or maybe invert it and have `cellar :any` and `cellar :relocatable` (the latter being the current behaviour, the prior not needing `install_name_tool`?